drake/systems/plants/RigidBodySystem.cpp

```diff
@@ -198,22 +194,19 @@ RigidBodySystem::StateVector<double> RigidBodySystem::dynamics(
   return dot;
 }
 
-RigidBodySystem::OutputVector<double> RigidBodySystem::output(
-    const double& t, const RigidBodySystem::StateVector<double>& x,
-    const RigidBodySystem::InputVector<double>& u) const {
-  if (sensors.size() == 0)
-    return x;
-  else {
-    auto kinsol = tree->doKinematics(x.topRows(tree->num_positions),
-                                     x.bottomRows(tree->num_velocities));
-    Eigen::VectorXd y(getNumOutputs());
-    int index = 0;
-    for (const auto& s : sensors) {
-      y.segment(index, s->getNumOutputs()) = s->output(t, kinsol);
-      index += s->getNumOutputs();
-    }
-    return y;
+RigidBodySystem::OutputVector<double> RigidBodySystem::output(const double& t,
+                                                              const RigidBodySystem::StateVector<double>& x,
+                                                              const RigidBodySystem::InputVector<double>& u) const
```

Comment:
i don't see anywhere where you set the direct feedthrough flag to true if you have an accelerometer.